### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by anjakammer**
_work in progress_

Development of the new Release Candidate 2023-01
by @sw4ma-marhai and @anjakammer

### CODE REVIEW

File: docs/02-Common-Architecture-Concepts/02-learning-goals.adoc [hunk 1]
```diff
@@ -2,39 +2,51 @@
...
 // tag::DE[]
 [[LZ-2-1]]
-==== LZ 2-1: Modularisierung und Kommunikation
-* verstehen die Ansätze moderner Architekturen (speziell Microservices, Self-contained Systems) und die Gründe, die zu diesen Lösungen führen
-* verstehen die unterschiedlichen Möglichkeiten der Kommunikation bzw. des Datenaustauschs der Services untereinander, zum Beispiel:
-** Event Sourcing
-** Messaging Middleware
-** RSS-Feeds
-* kennen die Auswirkungen einer Shared-Something/Nothing-Architektur, die insbesondere durch eine Skalierung über Container entsteht
+==== LZ 2-1: Situativ passende Kommunikations-Technologien verteilter Anwendungen auswählen
```
Comment (anjakammer) [inline docs/02-Common-Architecture-Concepts/02-learning-goals.adoc]:
```suggestion
==== LZ 2-1: Situativ passende Kommunikations-Konzepte verteilter Anwendungen auswählen
```

File: docs/02-Common-Architecture-Concepts/02-learning-goals.adoc [hunk 2]
```diff
@@ -2,39 +2,51 @@
...
+* Event Sourcing
+* Messaging Middleware
+* HTTP Feeds
+
+Softwarearchitekt:innen kennen die Auswirkungen einer Shared-Something/Nothing-Architektur, die insbesondere durch eine Skalierung über Container entsteht.
 
 [[LZ-2-2]]
-==== LZ 2-2: Modularisierung und Integration
-verstehen, wie man mehrere Services über unterschiedliche Wege zu einer Applikation integrieren kann, zum Beispiel durch Integration über:
+==== LZ 2-2: Situativ passende Integrations-Technologien verteilter Anwendungen auswählen
```
Comment (anjakammer) [inline docs/02-Common-Architecture-Concepts/02-learning-goals.adoc]:
```suggestion
==== LZ 2-2: Situativ passende Integrations-Konzepte verteilter Anwendungen auswählen
```

**Review by sw4ma-marhai — COMMENTED** [review]
Reviewed

**Review by sw4ma-marhai — APPROVED** [review]
Approved

**Pull request merged**